metricbeat/docs/modules/rabbitmq.asciidoc

```diff
@@ -21,7 +21,7 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: rabbitmq
-  metricsets: ["node", "queue", "connection"]
+  metricsets: ["node", "queue", "connection", "exchange"]
```

Comment:
Reminds me we should sort metricsets alphabetical (I can do that in a follow up PR).